src/modules/commander/Commander.cpp

```diff
@@ -3997,11 +3997,11 @@ void Commander::data_link_check(bool &status_changed)
 			}
 
 			if (_datalink_last_status_avoidance_system == telemetry_status_s::MAV_STATE_CRITICAL) {
-				mavlink_log_info(&mavlink_log_pub, "Avoidance system timeout");
+				mavlink_log_info(&mavlink_log_pub, "Avoidance system timed out");
 			}
 
 			if (_datalink_last_status_avoidance_system == telemetry_status_s::MAV_STATE_FLIGHT_TERMINATION) {
-				mavlink_log_critical(&mavlink_log_pub, "Avoidance system abort");
+				mavlink_log_critical(&mavlink_log_pub, "Avoidance system aborted");
```

Comment:
@mrivi I assume you know this: What's the meaning of `aborted` here? Is autonomous flight terminated and the flight controller stops listening to obstacle avoidance? Or is something wrong with the obstacle avoidance? I assume it's the former and not the latter. :)

Reply:
It means that there is something wrong with the avoidance and the vehicle doesn't accept anymore any setpoint from the OA system

Reply:
Should we then say "Avoidance system rejected" ?

Reply:
Fine for me